assets/js/admin-settings/index.jsx

```diff
@@ -0,0 +1,86 @@
+/* global localizedData */
+import {
+  Button,
+  SelectControl,
+  TextControl,
+  ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect, useState } from 'react';
+import useSiteOptions from '../services/hooks/use-site-options';
+import Rule from './rule';
+
+
+const AdminSettings = () => {
+  const [{ loading, saving, settings }, setOptions] = useSiteOptions();
+
+  /**
+   * Helper function for saving the in-memory settings to the server.
+   */
+   const saveSettings = (rule, ruleIndex) => {
+    // Remove old rule.
+    let newVal = settings.apple_news_automation ?? [];
+    if (ruleIndex) {
+      newVal = settings.apple_news_automation.filter((x, index) => index !== ruleIndex);
+    }
+    // Add updated rule.
+    newVal.push(rule);
+    const next = { ...settings, apple_news_automation: newVal };
+
+    // Enforce some defaults prior to save.
+    // Request will 500 when site_logo === null.
+    next.site_logo = next.site_logo ?? 0;
```

Comment:
See if this is still the case. Could have been a bug in an older version of WP.

Reply:
Is still the case, get a 500 when site_logo is null.